site/docs/v1/tech/installation-guide/securing.adoc

```diff
@@ -37,6 +37,22 @@ FusionAuth App to make requests to the search index.
 To access the FusionAuth UI you'll need to open port `9011`. If you have more than one instance of FusionAuth installed you will need
 to ensure that each instance of FusionAuth Backend can communicate on port `9011`.
 
+== Database Access
+
+FusionAuth uses a database to store almost all system data and configuration. This database contains extremely sensitive user information such as PII and password hashes. 
+
+Ensure that access to this database is limited to a single database user, and that the credentials of that user are controlled properly.
+
+In addition, access to backups of this database should be limited and backups permanently deleted when no longer of use.
+
+== Configuration Information
+
+The FusionAuth App is link:/docs/v1/tech/reference/configuration/[configured] using system properties, environment variables or a configuration file. This configuration includes sensitive information such as database credentials. Ensure you properly secure this configuration. For example:
+
+* If using a file, ensure the file is only readable by the user FusionAuth is running as.
+* If using environment variables, ensure they are stored safely and minimally accessible.
+* If using system properties, avoid setting them on the command line where they could be visible to other users on the system.
+
 == Custom Certificate Authority
 
 FusionAuth ships with the default `cacerts` file from OpenJDK, which contains a list of root certificate authorities. There are times when you need to add a certificate authority to the this file. This might occur if:
```

Comment:
```suggestion
FusionAuth ships with the default `cacerts` file from OpenJDK, which contains a list of root certificate authorities. There are times when you need to add a certificate authority to this file. This might occur if:
```

Reply:
Unrelated, but might as well fix this typo :)